internal/templates/internal/secrets.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'internal/base.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "internal/css/secrets.css" %}"/>
+    <script defer src="{% static 'internal/js/secrets.js' %}"></script>
+{% endblock %}
+
+{% block body %}
+    <div class="contentbox ui container">
+        <h1>
+            {% trans "Internal secrets" %}
+            {% if perms.internal.add_secret %}
+                <a href="{% url 'create-secret' %}">
+                    <i class="ui green plus icon"></i>
+                </a>
+            {% endif %}
+        </h1>
+        {% for secret in secrets %}
+            <h3>
+                {{ secret.title }}
+                {% if perms.internal.change_secret %}
+                    <a href="{% url 'edit-secret' secret.pk %}">
+                        <i class="ui yellow pencil icon"></i>
+                    </a>
+                {% endif %}
+                {% if perms.internal.delete_secret %}
+                    <a class="delete confirm" href="{% url 'delete-secret' secret.pk %}">
+                        <i class="red trash icon"></i>
+                    </a>
+                {% endif %}
+                <button class="secret-button ui red button" data-secret-id="secret-{{ secret.pk }}">
+                    {% trans "Press if you're alone" %}
+                </button>
+            </h3>
+            <div id="secret-{{ secret.pk }}" class="hidden">
+                {{ secret.content|safe }}
+                <p>{% trans "Last edited:" %} {{ secret.last_modified|safe }}</p>
```

Comment:
Maybe change the format of the timestamp, if possible. Right now it will look like  e.g. "2020-11-19 13:28:55.865926+00:00"

Reply:
It also shows the time in GMT, and not GMT+1 (which is our timezone)

Reply:
I agree 👍 You don't need the `safe` filter for inserting a datetime, and you can use the `date` filter for formatting the datetime string 🙂 For example:
```suggestion
                <p>{% trans "Last edited:" %} {{ secret.last_modified|date:"d.m.Y H:i" }}</p>
```
(you can look up the format characters in the [documentation on template filters](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#date))